frontend/cypress/integration/admin/admin-smoke-tests.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  aliasMutation,
+  aliasQuery,
+  setUpCommonGraphqlIntercepts,
+} from "../../support/graphql-test-utils";
+import { getInputByLabel } from "../../support/helpers";
+
+describe("Admin Smoke Tests", () => {
+  const loginViaUI = (role) => {
+    cy.fixture("users.json").then((users) => {
+      const user = users[role];
+      cy.get("input[name=username]").type(user.email);
+    });
+    cy.findByText("Sign-in").click();
+  };
+
+  beforeEach(() => {
+    // common requests
+    setUpCommonGraphqlIntercepts();
+    // page specific requests
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "AllUsers");
+      aliasQuery(req, "User");
+      aliasMutation(req, "UpdateUserAsAdmin");
+    });
```

Comment:
Curious your thoughts here -- my use of "common requests" is just a convenience I made up, thinking it would make it clearer to set up and see them all in one place. Intercepts seem pretty lightweight, so it didn't seem to impose a cost

My sense if that we should just keep it simple and either:
1. set them up for each test file and have no "common" intercepts (how the docs show it), or
2. store them all in one place (`setUpCommonGraphqlIntercepts()`) and avoiding thinking about "common" vs page-specific

I don't have a preference between those, but don't think we should do anything in-between :)

Reply:
I think I'd prefer option 1.  Each page tends to have its own set of customized requests and with the alias* functions you created it's really not much code to register them.  I'll try removing the common intercepts function.

Reply:
Thanks for removing it from the other specs too!